tests/beta/test_default_tensor.py

```diff
@@ -1705,7 +1706,7 @@ def test_hermitian(self, theta, phi, varphi, monkeypatch, rep, tol):
 
         with monkeypatch.context() as m:
             m.setattr("numpy.random.choice", lambda x, y, p: (x, p))
-            s1, p = dev.sample(["PauliZ", "Hermitian"], [[0], [1, 2]], [[], [A]])
+            s1, p = dev.sample(["PauliZ", "Hermitian"], [Wires([0]), Wires([1, 2])], [[], [A]])
 
         # s1 should only contain the eigenvalues of
         # the hermitian matrix tensor product Z
```

Comment:
Github highlighting really makes it easy to review a changes like the ones in this file easily :sweat_smile: